docker-compose.yml

```diff
@@ -89,9 +89,10 @@ services:
       - ./sites/${RICHIE_SITE:-funmooc}:/app
 
   terraform:
-    image: hashicorp/terraform:light
+    image: hashicorp/terraform
```

Comment:
Why not pinning a release instead?

Reply:
Because I like to be forced to upgrade. Terraform upgrades are usually very easy and we would never upgrade if we pinned it :man_shrugging:

Reply:
:thinking: ok then.

Reply:
well... we will always use the latest version available... no going back. I'm ok to pin if you think it's problematic but I'm afraid we will never upgrade.

Reply:
We should find a way to docker pull the latest version available ? Maybe doint it in one of the `bin/*` script ? Be cause after a first bootstrap we will never update it.

Reply:
Maybe we should force latest image pulling by adding `docker-compose pull terraform` some where in our `Makefile` or `bin/*` scripts?

The best solution would have been a pull policy in the `docker-compose` file, but it has not been implemented yet.

Reply:
@lunika @jmaupetit I added a fixup to use docker-compose also for the terraform state script and made sure that the latest images are always up-to-date before running terraform.